### ISSUE TITLE
We are not pushing workflows by dependency 

### ISSUE THREAD

**Issue by bryanboza** [dotCMS/core#13881]
As in previous versions, when you push some content type this should include the workflow by dependency 

## Expected Behavior
We need include the workflow by dependency in the bundle 

## Current Behavior
We are not pushing the workflow, just the content type 

## Steps to Reproduce (for bugs)
1. Login as admin
2. Create a new workflow 
3. Create a new content type and assign the created workflow 
4. Push publish this content type 

## Your Environment
Tested in the last master code // Postgres // FF

**Comment by oswaldogallango** [comment]
Hi @bryanboza I did some test and the workflow is being pushed. I can't reproduce your issue.

First I create two different workflows and two content types, then  I associated one workflow to each to each content type. I pushed one of the content types and the workflow was pushed. then I created a content for the other content type and pushed the contentlet, in that case also the workflow was pushed to the receiver. 

Then I created two different workflows and one content type, I associated both workflows to the content type and pushed the content type, same thing the workflow where pushed, then, I edited the workflows associated to the content type and push again the changes on the associated workflow where done.
